clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4810,7 +4810,68 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     }
   }
 
+#ifdef FLANG_LLVM_EXTENSIONS
```

Comment:
We have replaced FLANG_LLVM_EXTENSIONS with ENABLE_CLASSIC_FLANG in 56f27ac. You probably shouldn't undo that change.

Reply:
Somehow I left out the commit, thanks for mention it

Reply:
Fixed.